### ISSUE TITLE
Crash when clicking a user's profile

### ISSUE THREAD

**Issue by SamboyCoding** [WolfPlugs/GlobalBadges#2]
Reliably, when I click on one of my friend's profiles, discord crashes with this in the console:
```
DOMException: Blocked a frame with origin "https://canary.discord.com" from accessing a cross-origin frame.
    at replugged://renderer/:285:101
    at Array.every (<anonymous>)
    at replugged://renderer/:285:86
    at Array.find (<anonymous>)
    at getExportsForProps (replugged://renderer/:284:35)
    at replugged://renderer/:432:30
    at Array.find (<anonymous>)
    at getModule (replugged://renderer/:304:104)
    at Object.getByProps (replugged://renderer/:377:124)
    at replugged://plugin/dev.wolfplugs.globalbadges.asar/renderer.js?t=1672424544928}:398:48
```

**Comment by Wolfkid200444** [comment]
I cant reproduce

Is this a constant issue?

**Comment by SamboyCoding** [comment]
Yes, at least it was at time of reporting - I've since removed the plugin

**Comment by Wolfkid200444** [comment]
This bug should been fixed on version 1.0.1
